src/library/scala/StringContext.scala

```diff
@@ -138,7 +138,7 @@ case class StringContext(parts: String*) {
    *
    *  ''Note:'' Even when using the raw interpolator, Scala will process Unicode escapes.
    *  Unicode processing in the raw interpolator is deprecated as of scala 2.13.2 and
-   *  will be removed in scala 2.14
+   *  will be removed in the future
```

Comment:
:joy:

Reply:
You missed https://github.com/scala/scala/pull/8812/files#diff-644e3eed0afa90457d8cc36c6aedf87b where I avoided the irony of using "future" in Future.scala 😄